README.md

```diff
@@ -128,6 +128,7 @@ API will return these responses on certain error conditions:
  - 404 when deviceid isn't found
  - 480 when phone number/NFC id/mxid is not found at all
  - 481 when phone number/NFC id/mxid is found within member but has no access rights
+   - 481 responses will also contain basic user data for example executing proper procedures and admin logs
```

Comment:
Both 481 lines can be merged on the same line (now it looks like one of the 481 lines might be a typo or something)